microsoft-edge/webview2/concepts/power-user-faq.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Power User FAQ
+description: Answers to some frequently asked questions about WebView2 from non-developers who want to learn more about it.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 03/29/2023
+---
+# WebView2 Power User FAQ
+Most of our documentation is intended for developers who want to use WebView2 in their applications. This FAQ is intended for a normal person looking to understand what this `msedgewebview2.exe` application is on your computer. If we don't have your question here, feel free to hit "Feedback" in the top-right corner, or open a specific issue [on our GitHub](https://github.com/MicrosoftDocs/edge-developer/issues) and we'll see if we can add it.

```

Comment:
I'm not 100% sure about "normal person". "layperson" was suggested, but sounds similarly off?

Reply:
fixed locally
WebView2 end-user FAQ